DataAccess/Public/Include/DataAccess/IRepository.hpp

```diff
@@ -14,84 +15,94 @@ namespace DataAccess
 class IRepository
 {
 public:
+
     /**
-     * @brief Draft method for getting all channel list from the server.
-     * @return channels list as vector of ChannelInfo.
-     */
+    * @brief draft method for getting all channel list from the server.
+    * @return channels list as vector of ChannelInfo.
+    */
```

Comment:
Why did you change letter? This is the beginning of sentence in doxygen documentation.
![image](https://user-images.githubusercontent.com/90044672/183893926-523c66d7-f3c5-4656-81ba-48782d40f8f5.png)